src/api/.dockerignore

```diff
@@ -1,6 +1,9 @@
 .env
+.env.sample
+.gitignore
 .prettierrc.json
+.prettierignore
+.eslintrc.json
 .eslintignore
-node_modules
```

Comment:
we should keep the `node_modules` folder to avoid copying it to the Docker image manually

Reply:
Have added.